app/models/spree/admin/main_menu/default_configuration_builder.rb

```diff
@@ -0,0 +1,263 @@
+          add_dashboard_item(root)
+          add_orders_section(root)
+          add_returns_section(root)
+          add_products_section(root)
+          add_stocks_section(root)
+          add_reports_item(root)
+          add_promotions_section(root)
+          add_users_item(root)
+          add_content_section(root)
+          add_integrations_section(root)
+          add_oauth_section(root)
+          add_settings_section(root)
+          root
+        end
+
+        private
+
+        def add_dashboard_item(root)
+          root.add(ItemBuilder.new('dashboard', admin_dashboard_path).
+            with_label_translation_key('admin.home').
+            with_icon_key('house-door-fill.svg').
+            with_match_path('/dashboard').
+            build)
+        end
+
+        def add_orders_section(root)
+          items = [
+            ItemBuilder.new('completed', admin_orders_path(q: { completed_at_not_null: '1' })).
+              with_label_translation_key('admin.orders.all_orders').
+              with_match_path(/completed_at_not_null%5D=1/).
+              build,
+            ItemBuilder.new('draft_orders', admin_orders_path(q: { completed_at_not_null: false, state_eq: :cart })).
+              with_label_translation_key('admin.orders.draft_orders').
+              with_match_path(/cart/).
+              build,
+            ItemBuilder.new('abandoned_checkouts', admin_orders_path(
+                                                     q: { completed_at_not_null: false, state_in: %i[address delivery payment confirm] }
+                                                   )).
+              with_label_translation_key('admin.orders.abandoned_checkouts').
+              with_match_path(/state_in/).
+              build
+          ]
+
+          section = SectionBuilder.new('orders', 'inbox-fill.svg').
+                    with_admin_ability_check(Spree::Order).
+                    with_items(items).
+                    build
+          root.add(section)
+        end
+
+        def add_returns_section(root)
+          items = [
+            ItemBuilder.new('return_authorizations', admin_return_authorizations_path).
+              with_manage_ability_check(Spree::ReturnAuthorization).
+              with_match_path('/return_authorizations').
+              build,
+            ItemBuilder.new('customer_returns', admin_customer_returns_path).
+              with_manage_ability_check(Spree::CustomerReturn).
+              with_match_path('/customer_returns').
+              build
+          ]
+
+          section = SectionBuilder.new('returns', 'reply-fill.svg').
+                    with_manage_ability_check(Spree::ReturnAuthorization, Spree::CustomerReturn).
+                    with_items(items).
+                    build
+          root.add(section)
+        end
+
+        def add_products_section(root)
+          items = [
+            ItemBuilder.new('products', admin_products_path).with_match_path('/products').build,
```

Comment:
Is there any advantage to using the builder here rather than simply using the `Item` constructor?

Reply:
More explicit interface, getting default values for e.g. translation key, and skipping unnecessary arguments in the constructor.